packages/validator/test/cli-validator/tests/option-handling.test.js

```diff
@@ -119,25 +119,6 @@ describe('cli tool - test option handling', function() {
 
     const statsSection = capturedText.findIndex(x => x.includes('statistics'));
 
-    /*
-      Total number of errors   : 5
-      Total number of warnings : 8
-
-      errors
-       1  (20%) : Every operation must have unique "operationId".
-       2  (40%) : PUT and POST operations with a body parameter must have a non-empty `consumes` field.
-       1  (20%) : Arrays MUST NOT be returned as the top-level structure in a response body.
-       1  (20%) : Property names must follow case convention
-
-      warnings
-       1  (13%) : Potentially unused definition has been detected.
-       2  (25%) : Operations must have a non-empty `operationId`.
-       2  (25%) : A tag is defined but never used
-       1  (13%) : Schema must have a non-empty description.
-       1  (13%) : Schema properties must have a description with content in it.
-       1  (13%) : A security scheme is defined but never used
-    */
```

Comment:
This had gotten out of date and is obnoxious to keep up to date, so I just removed it

Reply:
Agreed, much easier to just print out the text via console.log temporarily.